File: DSA/SakshamDwivedi_590013147/Day15/D15P1.py
#Solution Function
def sort_arr(arr):
    return sorted(arr)

#Fixed Input
print(sort_arr([11, 4, 7, 14, 9, 10, 1]))

#Custom Input
arr = list(int(x) for x in input("Enter the array : ").split())
print(sort_arr(arr))

File: DSA/SakshamDwivedi_590013147/Day15/D15P1.c
#include <stdio.h>
#include <stdlib.h>

int arr[100];
int n;

int compare(const void *a, const void *b) {
    return (*(int *)a - *(int *)b);
}

int main() {

    printf("Enter number of elements: ");
    scanf("%d", &n);

    printf("Enter the elements :");
    for(int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    qsort(arr, n, sizeof(int), compare);

    printf("Sorted array: ");
    for(int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }
}

